clang/test/dpct/cache_remove.cu

```diff
@@ -0,0 +1,50 @@
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none -out-root %T/cache_remove %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck --input-file %T/cache_remove/cache_remove.dp.cpp --match-full-lines %s
+#include <cuda_runtime.h>
+
+__global__ void kernel()
+{
+  return;
+}
+
+int main()
+{
+  cudaStream_t stream;
+  const int pre_transform_tensor_size =
+      8 * 8 * 8;
+  const int transformed_tensor_size = pre_transform_tensor_size * 36 / 16;
+  const int res_block_mem =
+      transformed_tensor_size * 2 + pre_transform_tensor_size;
+  float *aptr;
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaLaunchAttributeValue is not supported.
+  cudaStreamAttrValue stream_attribute = {};
+  stream_attribute.accessPolicyWindow.base_ptr = aptr;
+  stream_attribute.accessPolicyWindow.num_bytes = res_block_mem;
+  stream_attribute.accessPolicyWindow.hitRatio = 1.0f;
+  stream_attribute.accessPolicyWindow.hitProp = cudaAccessPropertyPersisting;
+  stream_attribute.accessPolicyWindow.missProp = cudaAccessPropertyStreaming;
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaStreamSetAttribute is not supported.
+  cudaStreamSetAttribute(
+      stream, cudaLaunchAttributeIgnore, &stream_attribute);
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cudaStreamSetAttribute was removed because SYCL currently does not support setting cache config on devices.
+  cudaStreamSetAttribute(
+      stream, cudaStreamAttributeAccessPolicyWindow, &stream_attribute);
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaStreamGetAttribute is not supported.
+  cudaStreamGetAttribute(
+      stream, cudaLaunchAttributeIgnore, &stream_attribute);
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cudaStreamGetAttribute was removed because SYCL currently does not support setting cache config on devices.
+  cudaStreamGetAttribute(
+      stream, cudaStreamAttributeAccessPolicyWindow, &stream_attribute);
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cudaCtxResetPersistingL2Cache was removed because SYCL currently does not support setting cache config on devices.
+  cudaCtxResetPersistingL2Cache();
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cuCtxResetPersistingL2Cache was removed because SYCL currently does not support setting cache config on devices.
+  cuCtxResetPersistingL2Cache();
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cudaFuncSetAttribute was removed because SYCL currently does not support configuring shared emory on devices.
+  cudaFuncSetAttribute(kernel, cudaFuncAttributeMaxDynamicSharedMemorySize, 8 * 8 * 8);
+  // CHECK: DPCT1026:{{[0-9]+}}: The call to cudaFuncSetAttribute was removed because SYCL currently does not support configuring shared emory on devices.
+  cudaFuncSetAttribute(kernel, cudaFuncAttributePreferredSharedMemoryCarveout, 8 * 8 * 8);
+  // CHECK: DPCT1007:{{[0-9]+}}: Migration of cudaFuncSetAttribute is not supported.
```

Comment:
why some 1007 and other 1026?

Reply:
> why some 1007 and other 1026?
It depends on the usage of the function call. If function call be used for setting cache or shared memory config, we will report 1026 or 1027. In other case, we consider it as an unprocessed CUDA API call, report 1007 warning.

Reply:
Fixed, all of them will be remove and report 1026